sites/platform/src/administration/web/_index.md

```diff
@@ -8,7 +8,7 @@ description: |
 
 {{% description %}}
 
-![Overview of projects in the Console](/images/management-console/all-projects.png "0.5")
+![Overview of projects in the Console](/images/management-console/all-projects-revised.png "0.5")
```

Comment:
l.17: link to the console needs to be vendorized

Reply:
Fixed!